### DISCUSSION TITLE
handbook/enterprise_security/devsecops/toolchains/iast/

### DISCUSSION THREAD

**Discussion by giscus** [SEC-CAFE/handbook#7] [General]
# handbook/enterprise_security/devsecops/toolchains/iast/

动态应用安全测试(DAST)列表

https://sec.cafe/handbook/enterprise_security/devsecops/toolchains/iast/

**Comment by user1430** [comment]
这是一个深坑，在中国的大型业务不推荐实践IAST，常规漏洞的驱逐上投入产出完全不成正比，除非你期望进行特性漏洞的扫描发现
